vendor/github.com/markbates/goth/providers/github/github.go

```diff
@@ -1,4 +1,4 @@
-// Package github implements the OAuth2 protocol for authenticating users through Github.
```

Comment:
The CI will complain about modifications to the vendor directory. To add changes please have the changes merged upstream, then we can merge the changes into the project.